src/Page/NotAlone.elm

```diff
@@ -29,6 +32,79 @@ update msg model =
 view : Model -> Html Msg
 view model =
     div []
-        [ div [] [ text "Not Alone page" ]
+        [ header []
+            [ h1 [ css [ pageHeading ] ] [ text "[cCc] You Are Not Alone" ]
+            ]
+        , div [ css [ grid ] ]
+            [ card "[cCc] A medium quote about a recognisable experience. A medium quote about a recognisable experience." "My Name" "22"
+            , card "[cCc] A short quote about a recognisable experience" "My Name" "33"
+            , card "[cCc] A medium quote about a recognisable experience. A medium quote about a recognisable experience." "My Name" "62"
+            , card "[cCc] A long quote about a recognisable experience. A long quote about a recognisable experience, A long quote about a recognisable experience." "My Name" "33"
+            , card "[cCc] A short quote about a recognisable experience" "My Name" "55"
+            , card "[cCc] A long quote about a recognisable experience. A long quote about a recognisable experience, A long quote about a recognisable experience." "My Name" "44"
+            ]
+        , div [] [ text "Not Alone page" ]
         , a [ href "definition" ] [ text "Find out more about Economic Abuse" ]
         ]
+
+
+grid : Style
+grid =
+    batch
+        [ displayFlex
+        , flexWrap wrap
+        , justifyContent spaceBetween
+        ]
+
+
+card : String -> String -> String -> Html msg
+card quote name age =
+    div
+        [ css
+            [ cardStyle
+            , withMedia [ only screen [ Media.minWidth (px 576) ] ]
+                [ flex3 zero zero desktopWidth ]
+            ]
+        ]
+        [ span [ css [ quoteStyle ] ] [ text quote ]
+        , span [ css [ details ] ] [ text ("- " ++ name ++ ", " ++ age) ]
+        ]
+
+
+desktopWidth =
```

Comment:
Should these be in `Theme.elm` ?

Reply:
Maybe with better names. They're specific to the card widths here but we might want some similar styling with the resources grid. Abstract and rename with that card if it works out that way?

Reply:
:+1: